spec/index.bs

```diff
@@ -3729,78 +3729,72 @@ dictionary GPUCommandEncoderDescriptor : GPUObjectDescriptorBase {
-    <strong>|this|:</strong> of type {{GPUCommandEncoder}}.
-
-    **Arguments:**
-        - {{GPURenderPassDescriptor}} |descriptor|
-
-    **Returns:** {{GPURenderPassEncoder}}
-
-    Begins encoding a render pass described by |descriptor|.
-
-    On the [=Device timeline=], the following steps occur:
-
-        - If the [$GPUCommandEncoder.beginRenderPass/beginRenderPass Valid Usage$] rules are met:
-
-            - Set |this|.{{GPUCommandEncoder/[[state]]}} to {{encoder state/encoding a render pass}}.
-
-            - For each |colorAttachment| in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}:
-
-                - The texture [=subresource=] seen by |colorAttachment|.{{GPURenderPassColorAttachmentDescriptor/attachment}}
-                    is considered to be used as {{GPUTextureUsage/OUTPUT_ATTACHMENT}} for the
-                    duration of the render pass.
-
-            - Let |depthStencilAttachment| be |descriptor|.{{GPURenderPassDescriptor/depthStencilAttachment}}.
-            - If |depthStencilAttachment| is not `null`:
-
-                - The texture [=subresource=] seen by |depthStencilAttachment|.{{GPURenderPassDepthStencilAttachmentDescriptor/attachment}}
-                    is considered to be used as {{GPUTextureUsage/OUTPUT_ATTACHMENT}} for the
-                    duration of the render pass.
-
-    Issue: specify the behavior of read-only depth/stencil
-
-    <div class=validusage dfn-for=GPUCommandEncoder.beginRenderPass>
-        <dfn abstract-op>beginRenderPass Valid Usage</dfn>
-
-        Given the argument {{GPURenderPassDescriptor}} |descriptor|, the following validation rules apply:
-
-          1. |this|.{{GPUCommandEncoder/[[state]]}} must be {{encoder state/open}}.
-          1. |descriptor| must meet the
-            [$GPURenderPassDescriptor/GPURenderPassDescriptor Valid Usage$] rules.
-
-    </div>
-</div>
+<dl dfn-type="method" dfn-for="GPUCommandEncoder">
+    : <dfn>beginRenderPass(descriptor)</dfn>
+    ::
 
-### <dfn method for=GPUCommandEncoder>beginComputePass(descriptor)</dfn> ### {#GPUCommandEncoder-beginComputePass}
+        Begins encoding a render pass described by |descriptor|.
 
-<div algorithm="GPUCommandEncoder.beginComputePass">
-    <strong>|this|:</strong> of type {{GPUCommandEncoder}}.
+        <div algorithm=GPUCommandEncoder.beginRenderPass>
+            **Called on:** {{GPUCommandEncoder}} |this|.
 
-    **Arguments:**
-        - {{GPUComputePassDescriptor}} |descriptor|
+            **Arguments:**
+            <ul dfn-type=argument dfn-for="GPUCommandEncoder/beginRenderPass(descriptor)">
+                - |descriptor|: {{GPURenderPassDescriptor}} <dfn>descriptor</dfn>
+            </ul
 
-    **Returns:** {{GPUComputePassEncoder}}
+            **Returns:** {{GPURenderPassEncoder}}
 
-    Begins encoding a compute pass described by |descriptor|.
+            - Issue the following steps on the [=Device timeline=] of |this|:
```

Comment:
optional nit: since there's only one item in this top-level list, it would probably make sense to dissolve it and save one indentation level.